nexus/src/external_api/http_entrypoints.rs

```diff
@@ -1868,6 +1870,28 @@ async fn hardware_sleds_get_sled(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/*
+ * Updates
+ */
+
+/**
+ * Refresh update metadata
+ */
```

Comment:
This seems like a good candidate for an addition to https://github.com/oxidecomputer/omicron/blob/main/tools/oxapi_demo , as a convenience